Data/Common/BaseEntity.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace LearnNet_CatalogService.Data.Common
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace LearnNet_CatalogService.Data.Common
 {
     public class BaseEntity
     {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
You should not use both ways of entity configuration at the same time.
Either do everything in OnModelCreating (preferably) or via attributes